include/items.php

```diff
@@ -97,7 +97,7 @@ function add_page_info_data(array $data, $no_photos = false)
 			/// @TODO make a positive list of allowed characters
 			$hashtag = str_replace([" ", "+", "/", ".", "#", "'", "’", "`", "(", ")", "„", "“"],
 						["", "", "", "", "", "", "", "", "", "", "", ""], $keyword);
-			$hashtags .= "#[url=" . System::baseUrl() . "/search?tag=" . rawurlencode($hashtag) . "]" . $hashtag . "[/url] ";
+			$hashtags .= "#[url=" . System::baseUrl() . "/search?tag=" . $hashtag . "]" . $hashtag . "[/url] ";
```

Comment:
Why have you removed the "rawurlencode" here? And possibly you could instead work with a positive list, like suggested in the ```@todo```?

Reply:
Because this URL isn't meant to be displayed in a HTML context, we're building a BBCode tag that doesn't require any specific encoding.

On top of that, we are completely discarding the URL in favor of a rebuilt URL based on the hashtag term when we eventually convert to HTML, and at that point we are URL encoding.

And it wasn't in the scope of this PR but no, we can't work with a positive list of characters because there are accents and different alphabets (Japanese, Chinese, Arabic, Cyrillic,...) to take into account.